cmd/crates/soroban-spec-typescript/src/lib.rs

```diff
@@ -293,35 +303,27 @@ pub fn entry_to_method_type(entry: &Entry) -> String {
 ",
             )
         }
-        Entry::ErrorEnum { doc, cases, .. } => {
-            let doc = doc_to_ts_doc(doc, None, 0);
-            let cases = cases
-                .iter()
-                .enumerate()
-                .map(|(i, c)| {
-                    if c.doc.is_empty() {
-                        format!(
-                            "{}  {}: {{message:\"{}\"}}",
-                            if i == 0 { "" } else { "\n" },
-                            c.value,
-                            c.name
-                        )
-                    } else {
-                        format!(
-                            "{}{}  {}: {{message:\"{}\"}}",
-                            if i == 0 { "" } else { "\n" },
-                            doc_to_ts_doc(&c.doc, None, 1),
-                            c.value,
-                            c.name
-                        )
-                    }
-                })
-                .join(",\n");
-            format!("{doc}export const Errors = {{\n{cases}\n}}")
+        Entry::ErrorEnum { .. } => {
+            // type filtered out before entry_to_method_type is called
```

Comment:
This lines confuses me.  I don't see where it is filtered.

Reply:
That's the whole `filter_map` thing starting on [line 125](https://github.com/stellar/stellar-cli/pull/2031/files#diff-81cd9dbf2c88f86e0c16d781e82b16a951ea8148d057e530aef67105a7df3ab3R125)

Reply:
I see that, but that is on a clone of the original `collected`. And later it seems that the original is used.

```rust
let (fns, other): (Vec<_>, Vec<_>) = collected
        .into_iter()
        .filter(|entry| !matches!(entry, Entry::Function { name, .. } if name.starts_with("__")))
        .partition(|entry| matches!(entry, Entry::Function { .. }))
```rust

Reply:
Why I hate clone!

Reply:
The referenced code has been removed and the comment has been updated. `ErrorEnum` logic here is still a no-op.